.eslintignore

```diff
@@ -1,5 +1,6 @@
 **/js/jquery/1.11/*
 **/js/jquery/1.12/*
```

Comment:
Still useful ?

Reply:
jquery 1.11 is not removed from code base